wormhole-connect/src/utils/events.ts

```diff
@@ -1,25 +1,48 @@
 import { BigNumber } from 'ethers';
 import {
-  ChainId,
   ChainName,
-  TokenId,
   EthContext,
   SuiContext,
   WormholeContext,
 } from '@wormhole-foundation/wormhole-connect-sdk';
-import { wh } from '../sdk';
+import { ParsedMessage, ParsedRelayerMessage, PaymentOption, wh } from '../sdk';
 import { fromNormalizedDecimals } from '.';
 import { CHAINS } from '../config';
 import { arrayify } from 'ethers/lib/utils.js';
+import { fetchGlobalTx, getEmitterAndSequence } from './vaa';
+
+export const fetchRedeemTx = async (
+  txData: ParsedMessage | ParsedRelayerMessage,
+): Promise<{ transactionHash: string } | null> => {
+  try {
```

Comment:
I think it might be best for readability to only wrap the `fetchGlobalTx` in the try/catch and avoid throwing the Error, which could be a `console.warn` instead.